apps/storybook/src/Select.test.ts

```diff
@@ -37,7 +39,13 @@ describe('Select', () => {
       cy.get('.iui-select-button').click();
       cy.compareSnapshot(`${testName} (Open)`);
 
-      if (testName.includes('Custom')) {
+      if (testName.includes('Multi')) {
+        cy.get('.iui-menu-item').first().click();
+        cy.get('.iui-menu-item').eq(1).click();
+        cy.compareSnapshot(`${testName} (Multi selected)`);
+      }
+
+      if (testName.includes('Custom') && !testName.includes('Multi')) {
```

Comment:
Don't you want just write `testName === 'Custom'`?

Reply:
ohh, haha 
Good we have reviewers